media.go

```diff
@@ -949,46 +949,40 @@ func (insta *Instagram) UploadPhoto(photo io.Reader, photoCaption string, qualit
 
 func (insta *Instagram) postPhoto(photo io.Reader, photoCaption string, quality int, filterType int, isSidecar bool) (map[string]interface{}, error) {
 	uploadID := time.Now().Unix()
-	photoName := fmt.Sprintf("pending_media_%d.jpg", uploadID)
-	var b bytes.Buffer
-	w := multipart.NewWriter(&b)
-	w.WriteField("upload_id", strconv.FormatInt(uploadID, 10))
-	w.WriteField("_uuid", insta.uuid)
-	w.WriteField("_csrftoken", insta.token)
-	var compression = map[string]interface{}{
-		"lib_name":    "jt",
-		"lib_version": "1.3.0",
-		"quality":     quality,
-	}
-	cBytes, _ := json.Marshal(compression)
-	w.WriteField("image_compression", toString(cBytes))
-	if isSidecar {
-		w.WriteField("is_sidecar", toString(1))
-	}
-	fw, err := w.CreateFormFile("photo", photoName)
+	rndNumber := rand.Intn(9999999999-1000000000) + 1000000000
```

Comment:
why this is needed?

Reply:
I add this random number to generate name of the upload (part of url) and then use as value of 'X-Entity-Name' header. I use the same range used by https://github.com/dilame/instagram-private-api/blob/master/src/repositories/upload.repository.ts#L20. When i listen instagram api call from app, the requested url for upload fit this generate string.